helix-term/src/commands/lsp.rs

```diff
@@ -79,6 +79,7 @@ impl ui::menu::Item for lsp::Location {
             // With the preallocation above and UTF-8 paths already, this closure will do one (1)
             // allocation, for `to_file_path`, else there will be two (2), with `to_string_lossy`.
             let mut write_path_to_res = || -> Option<()> {
+                // We don't convert to a `helix_core::Uri` here because we've already checked the scheme.
                 let path = self.uri.to_file_path().ok()?;
```

Comment:
NB: this will not necessarily be normalized. i suppose its not necessary since it's just for display but it would be good to note.